catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/GravitinoPaimonTable.java

```diff
@@ -137,21 +137,10 @@ private List<String> getPrimaryKeysFromIndexes(Index[] indexes) {
     if (indexes == null || indexes.length == 0) {
       return Collections.emptyList();
     }
-
     Preconditions.checkArgument(
         indexes.length == 1, "Paimon only supports no more than one Index.");
-
     Index primaryKeyIndex = indexes[0];
-    Arrays.stream(primaryKeyIndex.fieldNames())
```

Comment:
any reason to remove this? does Paimon support the nested column as the Primary key?